scss/bitstyles/ui/app-layouts.stories.mdx

```diff
@@ -0,0 +1,530 @@
+              </ul>
+            </div>
+          </header>
+          <div class="u-flex__grow-1 u-overflow--y u-grid u-gap-l u-grid--2-col@xl a-content a-content--full u-margin-0">
+            <div class="u-padding-l-y">
+              <form action="" method="POST" role="search" class="u-margin-m-bottom u-flex">
+                <h3 class="u-sr-only">Filter results</h3>
+                <ul class="o-ui-group u-flex a-list-reset">
+                  <li class="u-flex">
+                    <label for="search_users" class="u-sr-only">Filter users by email</label>
+                    <input type="search" id="search_users" class="o-ui-group__item u-round-0--tr u-round-0--br" placeholder="Filter by email…" />
+                  </li>
+                  <li class="u-flex u-relative">
+                    <button type="button" class="a-button a-button--ui u-round-0--tl u-round-0--bl">
+                      Filter
+                    </button>
+                  </li>
+                </ul>
+                <button class="a-button u-sr-only" type="submit">Apply filter</button>
+              </form>
+              <div class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative u-margin-xl-bottom">
+                <div class="u-absolute-center">
+                  To make the content easier to read, width is limited by the grid system. Other content follows below, except on xl screens, where other content is allowed adjacent
+                </div>
+              </div>
+            </div>
+            <div class="u-padding-l-y">
+              <h2>Further content</h2>
+              <ul class="a-list-reset u-grid u-grid--2-col@l u-grid--3-col@xl u-gap-l u-margin-xl-bottom">
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+                <li class="a-card u-bg--background u-aspect-ratio u-aspect-ratio-16-9 u-relative">
+                  <div class="u-absolute-center">Content block</div>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </main>
+      </div>
+    `}
+  </Story>
+</Canvas>
```

Comment:
TLDR: I trust you with this changes 😄. Thanks for attaching the image to the PR description :)